apps/frontend/src/libs/components/input/input.tsx

```diff
@@ -14,10 +14,11 @@ import styles from "./styles.module.css";
 type Properties<T extends FieldValues> = {
 	control: Control<T, null>;
 	errors?: FieldErrors<T>;
+	hasVisuallyHiddenLabel?: boolean;
 	iconName?: IconName;
 	label: string;
 	name: FieldPath<T>;
-	onIconClick?: () => void;
+	onIconClick?: (() => void) | undefined;
```

Comment:
Why is this needed?

Reply:
I think this is for hide/show password needed.